packages/plugins/rudder_plugin_interface/test/src/models/utils_test.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:test/test.dart';
+import 'package:rudder_sdk_flutter_platform_interface/src/utils.dart';
+
+void main() {
+  test('conversion of datestring object to string in format YY-mm-dd', () {
+    final DateTime date = DateTime(2020, 12, 12);
+    expect(Utils.toDateString(date), equals("20-12-12"));
+    final DateTime date2 = DateTime(2022, 7, 25);
+    expect(Utils.toDateString(date2), equals("22-07-25"));
+    final DateTime date3 = DateTime(2023, 1, 13);
```

Comment:
Add a case of leading zeroes for the day field too.

Reply:
added